plugins/scorecards/package.json

```diff
@@ -0,0 +1,52 @@
+{
+  "name": "@backstage/plugin-scorecards",
```

Comment:
I wonder if we should call this something a little more `tech-insights` specific?

Something like `plugin-tech-insights-scorecards`?

Reply:
It is addressed in the readme that this is a part of as a part of the Backstage Tech Insights feature. Reason for naming it simply 'plugin-scorecards' was because it seemed more straightforward for people who might not be 100% familiar with tech insights, hence they may not look for tech-insights-scorecards at first 🤷🏼‍♀️ However, I am also fine with renaming it :)  @benjdlambert

Reply:
I vote to rename too! I'm thinking, with the current naming it's "the scorecards plugin". Could also perhaps go in the actual plugin instead of a separate package I guess, or even in core-components - but having it separate is fine too. So yeah, a rename as proposed sounds good.

Reply:
Oh hang on. Let me revise that. The standard module naming pattern would be to name it `@backstage/plugin-tech-insights-react` because that's where people normally put shared frontend parts that other plugins also should be able to import, right?

@backstage/maintainers here :)